src/renderer/components/ft-share-button/ft-share-button.js

```diff
@@ -0,0 +1,87 @@
+import Vue from 'vue'
+
+import FtIconButton from '../ft-icon-button/ft-icon-button.vue'
+import FtButton from '../ft-button/ft-button.vue'
+
+export default Vue.extend({
+  name: 'FtShareButton',
+  components: {
+    'ft-icon-button': FtIconButton,
+    'ft-button': FtButton
+  },
+  props: {
+    id: {
+      type: String,
+      required: true
+    }
+  },
+  computed: {
+    invidiousInstance: function () {
+      return this.$store.getters.getInvidiousInstance
+    },
+
+    usingElectron: function () {
+      return this.$store.getters.getUsingElectron
+    },
+  },
+  methods: {
```

Comment:
There isn't a good spot to put this, so I'll just put this here.

Clicking on the buttons within this dropdown should close it once it is clicked.  The way to do this isn't very obvious so I can help you out on getting this done.  It's possible to call `toggleDropdown` from this component using [this guide](https://vuejs.org/v2/guide/components-edge-cases.html#Accessing-Child-Component-Instances-amp-Child-Elements).  Let me know if you have any problems with this.